web/files/files.go

```diff
@@ -307,13 +307,23 @@ func FileCopyHandler(c echo.Context) error {
 		return WrapVfsError(err)
 	}
 
+	copyName := c.QueryParam("Name")
+	if copyName == "" || filepath.Ext(copyName) != filepath.Ext(olddoc.DocName) {
+		copyName = fileCopyName(inst, olddoc.DocName)
+	}
+
+	newdoc := vfs.CreateFileDocCopy(olddoc, copyName)
+
+	if newDirID := c.QueryParam("dir_id"); newDirID != "" {
```

Comment:
s/dir_id/DirID/ ?

Reply:
Which route uses `dir_id` in the query-string?

Reply:
Not in the query-string but `PATCH /files/:file-id` accepts `dir_id` as an patch attribute. But if your intention is to have all query-string parameters be in PascalCase then it makes sense to use `DirId` instead (or `DirID` in this case).